templates/design_system/authorize.html

```diff
@@ -58,7 +58,7 @@ <h1>{{ application.name }} WILL BE ABLE TO:</h1>
       
                      <div class="ds-l-row">
                       <div class="ds-l-col--auto">
-                          <a class="ds-c-button  ds-c-button--outline ds-u-margin-top--1 ds-u-display--block ds-u-md-display--inline-block" href ="{% url 'do_not_approve' %}" >No, do not approve access</a>
```

Comment:
Can you delete the linked page as part of this change?

Reply:
Removed the template.

Reply:
@samgensburg-gov removed the endpoint too.